### ISSUE TITLE
Support for additional mounts

### ISSUE THREAD

**Issue by pniederlag** [influxdata/helm-charts#208]
We'd like to see a support for additional volumes/volumeMounts.

### Rationale

adding a ssl certificate that we need in order to connect to an external influxdb

### Inspiration

the [keycloak chart](https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/values.yaml) provides two optional variables that can be used for such purposes.

extraVolumes
extraVolumeMounts

Is this something you'd in general support and willing to accept a Pull-Request?

**Comment by rawkode** [comment]
@pniederlag Our Telegraf chart supports this. We'd welcome PRs that add this to other charts, where it makes sense.

https://github.com/influxdata/helm-charts/blob/12f45d788f0e0218c7357cdee57258ccc344b8cc/charts/telegraf/values.yaml#L28

**Comment by nanaboat** [comment]
@rawkode I will like to work on this.

**Comment by caleb15** [comment]
@rawkode the part you linked is commented out FYI

**Comment by rawkode** [comment]
It’s commented out, as there are no mounts by default. It’s there as an example of something that’s available

**Comment by caleb15** [comment]
Sorry, I'm still a newbie to helm. I thought values.yml listed the options available, but as I understand it now values.yml is just the default options.

**Comment by rawkode** [comment]
No problem. I’ll put together some videos and tutorials for consuming these random options. If you spot anything else that could use some examples, please let me know 😀

**Comment by caleb15** [comment]
Ah, I talked to my boss and there is still a issue I think. If I passed in a volume with a configmap I would have to create a configmap, but https://github.com/influxdata/helm-charts/blob/master/charts/telegraf/templates/configmap.yaml doesn't allow for the creation of custom configmaps passed in via values. I might submit a PR to add that. Or bundle my files in our own docker image and just pass that in :thinking:
